ansible_collections/arista/avd/examples/campus-fabric/group_vars/CONNECTED_ENDPOINTS.yml

```diff
@@ -1,9 +1,9 @@
 ---
-### group_vars/DC1_NETWORK_PORTS.yml
+### group_vars/CONNECTED_ENDPOINTS.yml
 
 port_profiles:
   - profile: PP-DOT1X
-    mode: "trunk phone"
+    mode: trunk phone
```

Comment:
why we remove quotes? I'd prefer a string with space to be quoted. Ideally all the strings. Even if not required, quoting all the string in YAML is the best practice.

Reply:
No, I'm not suggesting to quote all the strings in the example. But this one for sure.